helm/microservice/values.yaml

```diff
@@ -3,6 +3,8 @@
 
 replicaCount: 1
 
+namespace: go-api-boilerplate
```

Comment:
Why do we need that here ?

This chart should not be coupled to any namespace. to remove.

Reply:
It's been used in the deployment and service in helm/microservice and never defined.

For example:
`helm/microservice/templates/deployment.yaml`

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: {{ include "microservice.fullname" . }}
  {{- if hasKey .Values "namespace" }}
  namespace: {{ .Values.namespace }}
  {{- end }}
```

`helm/microservice/templates/service.yaml`

```
apiVersion: v1
kind: Service
metadata:
  name: {{ include "microservice.fullname" . }}
  {{- if hasKey .Values "namespace" }}
  namespace: {{ .Values.namespace }}
  {{- end }}
```